packages/server/src/ddApm.ts

```diff
@@ -3,5 +3,9 @@ import apm from "dd-trace"
 // enable APM if configured
 if (process.env.DD_APM_ENABLED) {
   console.log("Starting dd-trace")
-  apm.init()
+  apm.init({
+    // @ts-ignore for some reason dd-trace types don't include this options,
+    // even though it's spoken about in the docs.
+    debug: process.env.DD_ENV === "qa",
```

Comment:
Should we not use the node env settings?

Reply:
I was in two minds about this. I don't mind either way, personally, but decided to go with the `DD_ENV` here because it's datadog-specific code.

Reply:
I'm happy enough for it to work this way - we can change it if its a problem.